src/lib/components/workflow/metadata/metadata-events.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import MetadataDecoder from '$lib/components/event/metadata-decoder.svelte';
+  import WorkflowStatus from '$lib/components/workflow-status.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { groupEvents } from '$lib/models/event-groups';
+  import { fullEventHistory } from '$lib/stores/events';
+  import { workflowRun } from '$lib/stores/workflow-run';
+  const { workflow } = $derived($workflowRun);
+  const metadataGroups = $derived(
+    groupEvents(
+      $fullEventHistory,
+      'ascending',
+      workflow?.pendingActivities,
+      workflow?.pendingNexusOperations,
+    ).filter((group) => group.userMetadata?.summary),
+  );
+
+  $inspect('Metadata groups:', metadataGroups);
```

Comment:
🗑️

Reply:
Dang it... I searched for console logs at the end and forgot about inspect. Thank you!